libavcodec/vvc/dec.c

```diff
@@ -931,6 +983,7 @@ static int decode_nal_units(VVCContext *s, VVCFrameContext *fc, AVPacket *avpkt)
     s->last_eos = s->eos;
     s->eos = 0;
 
+    ff_vvc_reset_sei(&fc->sei);
```

Comment:
We can not reset SEI for every frame.  For example, film grain information will use by any following frames until it canceled.


> fg_characteristics_cancel_flag equal to 1 indicates that the SEI message cancels the persistence of any previous film
> grain characteristics SEI message in output order that applies to the current layer. fg_characteristics_cancel_flag equal to 0
> indicates that film grain modelling information follows